pkg/endpoint/endpoint.go

```diff
@@ -1794,7 +1798,10 @@ func (e *Endpoint) identityLabelsChanged(ctx context.Context, myChangeRev int) e
 	e.unlock()
 
 	if readyToRegenerate {
-		e.Regenerate(&regeneration.ExternalRegenerationMetadata{Reason: "updated security labels"})
+		e.Regenerate(&regeneration.ExternalRegenerationMetadata{
+			Reason:            "updated security labels",
+			RegenerationLevel: regeneration.RegenerateWithoutDatapath, // Needs datapath load??
```

Comment:
Identity is a static variable substituted in the ELF:

https://github.com/cilium/cilium/blob/2845ccdfcdbb53a80998a9ea06ddb5f8ff1134ca/pkg/datapath/loader/template.go#L217

So we need to do an ELF rewrite here.

```suggestion
			RegenerationLevel: regeneration.RegenerateWithDatapathRewrite,
```

Reply:
Refactored with this change.